src/components/PasswordInput.js

```diff
@@ -1,5 +1,29 @@
 import React from "react";
+import styled from "@emotion/styled";
 
-export default function PasswordInput({ value }) {
-  return <input value={value} />;
+const Input = styled.input`
+  background-color: hsl(
+    ${props =>
+      props.value?.length <= 10
```

Comment:
Multiple `?` are different to read. Maybe a function will help to make it more readable?
the `?` behind wouldn't be required if the default state is `""` instead of `null`